dojo/importers/utils.py

```diff
@@ -1,30 +1,10 @@
 from dojo.utils import max_safe
-from dojo.tools.factory import get_parser
 from dojo.models import IMPORT_CLOSED_FINDING, IMPORT_CREATED_FINDING, IMPORT_REACTIVATED_FINDING, Test_Import, Test_Import_Finding_Action
 import logging
 
 logger = logging.getLogger(__name__)
 
 
-def parse_findings(scan, test, active, verified, scan_type):
-    try:
-        parser = get_parser(scan_type)
-        parsed_findings = parser.get_findings(scan, test)
-        return parsed_findings
-    except SyntaxError as se:
```

Comment:
This exception handling seems to be gone completely now, do we really not need it? Errors in parsers happen often, eg. when they change their format. Where will errors be logged now?

Reply:
The exception is still correctly handled.
Ex:
![image](https://user-images.githubusercontent.com/1694940/134778935-4f83c245-25cd-41b3-8430-709767a0e6d2.png)

The log are still there:
```
uwsgi_1         | [25/Sep/2021 16:33:51] DEBUG [dojo.importers.importer.importer:304] IMPORT_SCAN parser v2: Create Test and parse findings
uwsgi_1         | [25/Sep/2021 16:33:51] ERROR [dojo.engagement.views:614] name 'b' is not defined
uwsgi_1         | Traceback (most recent call last):
uwsgi_1         |   File "/app/./dojo/engagement/views.py", line 599, in import_scan_results
uwsgi_1         |     test, finding_count, closed_finding_count = importer.import_scan(scan, scan_type, engagement, user, environment, active=active, verified=verified, tags=tags,
uwsgi_1         |   File "/app/./dojo/importers/importer/importer.py", line 305, in import_scan
uwsgi_1         |     tests = parser.get_tests(scan_type, scan)
uwsgi_1         |   File "/app/./dojo/tools/sarif/parser.py", line 41, in get_tests
uwsgi_1         |     a = b / 0
uwsgi_1         | NameError: name 'b' is not defined
uwsgi_1         | [25/Sep/2021 16:33:51] DEBUG [dojo.jira_link.helper:41] JIRA is disabled, not doing anything
uwsgi_1         | [pid: 149|app: 0|req: 17/21] 172.21.0.1 () {60 vars in 1378 bytes} [Sat Sep 25 16:33:50 2021] POST /product/10/import_scan_results => generated 71016 by
```
The previous version just logged and raised again the same exception
Not sure why it was required.

Reply:
I'm testing the two version but honestly I prefer when we have less code.
The function without the try/catch was just 2 lines. Not very useful.

Reply:
I checked, the views and the serializers are catching the exception and care about logging. So we don't need the code you deleted.